Mathlib/Tactic/NormNum/Test.lean

```diff
@@ -0,0 +1,17 @@
+import Mathlib.Tactic.NormNum.Eq
+import Mathlib.Algebra.Field.ZMod
+
+example {K : Type*} [Field K] [CharP K 37] : ((3/4 : ℚ) + (5 : ℤ) : K) = 15 := by
+  norm_num (char := [3, 37])
```

Comment:
Doesn't the fact that the `p` argument of `CharP` is a `semiOutParam` mean that you can just infer the characteristic?

Reply:
Good point. But why won't it just find `instance charP : CharP R (ringChar R)` every time then?

Reply:
Ah, I think I might be mistaken, and that only `outParam` would work here